packages/training-link/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "1.3.3",
   "author": "Andrew Bauer <andrew.bauer@availity.com>",
   "description": "Component for allowing link out to training in the Header component",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
```

Comment:
It is ok to have these pointing to the same file? I just had to read up on this module field a bit and I'm still a little confused about it.

Reply:
I'll double check, but I'm pretty sure I had to add it for tests or storybook.